app/controllers/books_controller.rb

```diff
@@ -4,7 +4,14 @@ class BooksController < ApplicationController
   before_action :set_book, only: %i[edit update destroy]
   before_action :require_admin_or_mentor_login, except: %i[index]
 
-  def index; end
+  def index
+    books = Book.all
+    books = params[:practice_id].present? ? books.joins(:practices).where(practices: { id: params[:practice_id] }) : books
+    @books = books
+             .with_attached_cover
+             .includes(:practices)
+             .order(updated_at: :desc, id: :desc)
```

Comment:
```suggestion
@books = books
  .with_attached_cover
  .includes(:practices)
  .order(updated_at: :desc, id: :desc)
```

でよかった気がします〜

Reply:
@komagata 
こちら、この位置で揃えないとRubocopの`Layout/MultilineMethodCallIndentation`というルールに引っかかってしまうのですが、大丈夫でしょうか？👀

Reply:
すみません、上記だとrubocopに引っかかるんですね。
下記はいかがでしょうか。
```suggestion
    @books = books.with_attached_cover
                  .includes(:practices)
                  .order(updated_at: :desc, id: :desc)
```

Reply:
@komagata 
こちらの書き方で大丈夫でした🙆